src/main/java/lotto/LottoResult.java

```diff
@@ -0,0 +1,42 @@
+package lotto;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LottoResult {
+    private static final int PERCENT = 100;
+
+    public double calculateEarningRate(List<Rank> ranks, int purchaseMoney) {
+        Map<Rank, Integer> result = completeResult(ranks);
+        Rank[] ranksSpreadInOrder = Rank.values();
+        double profit = 0;
+
+        for (Rank rank : ranksSpreadInOrder) {
+            profit += result.get(rank) * rank.getPrize();
+        }
+
+        return profit / purchaseMoney * PERCENT;
+    }
+
+    public Map<Rank, Integer> completeResult(List<Rank> ranks) {
+        Map<Rank, Integer> result = initResult();
+
+        for (Rank rank : ranks) {
+            result.put(rank, result.getOrDefault(rank, 0) + 1);
+        }
+
+        return result;
+    }
+
+    private Map<Rank, Integer> initResult() {
+        Map<Rank, Integer> result = new HashMap<>();
+        result.put(Rank.THREE_MATCH, 0);
+        result.put(Rank.FOUR_MATCH, 0);
+        result.put(Rank.FIVE_MATCH, 0);
+        result.put(Rank.FIVE_MATCH_AND_BONUS, 0);
+        result.put(Rank.SIX_MATCH, 0);
+
+        return result;
```

Comment:
(이렇게 바꾸면 좋을 것 같아요)

```suggestion
        return Arrays.stream(Rank.values())
            .collect(toMap(Function.identity(), (value) -> 0));
```
이렇게 스트림 API를 활용하면 모든 요소를 0으로 쉽게 초기화 할 수 있어요:)

Reply:
훨씬 깔끔하네요!!